exercises/rail-fence-cipher/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+pub struct RailFence;
+
+impl RailFence {
+    pub fn new(rails: u32) -> RailFence {
```

Comment:
I think I understand why the API is this way. It subtly encourages computing any calculations specific to the number of rails only once. That seems reasonable.

Behaviour might be unpredictable if someone passes `0`, but we are not testing the behaviour so I don't see a need for a change.